src/packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -139,17 +141,7 @@ Object.assign(exports, Prisma)
  * In order to make \`ncc\` and \`@vercel/nft\` happy.
  * The process.cwd() annotation is only needed for https://github.com/vercel/vercel/tree/master/packages/now-next
 **/
-${
-  this.options.platforms
-    ? this.options.platforms
-        .map(
-          (p) => `path.join(__dirname, 'query-engine-${p}');
-path.join(process.cwd(), './${path.join(cwdDirname, `query-engine-${p}`)}');
-`,
-        )
-        .join('\n')
-    : ''
-}
+${buildNFTEngineAnnotations(this.options.generator?.previewFeatures.includes('napi') ?? false, this.options.platforms as Platform[], cwdDirname)}
```

Comment:
```suggestion
${buildNFTEngineAnnotations(this.options.generator?.previewFeatures?.includes('napi') ?? false, this.options.platforms as Platform[], cwdDirname)}
```

Since it could be undefined it could error with `Uncaught TypeError: can't access property "includes", previewFeatures is undefined`